nestor_api/utils/dict.py

```diff
@@ -15,8 +15,7 @@ def deep_merge(destination: dict, source: dict) -> dict:
 
     def _deep_merge_rec(dest, src):
         for key in src:
-            # pylint: disable=bad-continuation
-            if key in dest and isinstance(dest.get(key), dict) and isinstance(src[key], dict):
+            if key in dest and isinstance(dest[key], dict) and isinstance(src[key], dict):
```

Comment:
```suggestion
            if isinstance(dest.get(key), dict) and isinstance(src[key], dict):
```
If key is not in dict, it will return `None`, no need for the extra check 😉